package.json

```diff
@@ -35,6 +35,8 @@
   },
   "scripts": {
     "start": "webpack-dev-server --inline --hot --content-base dist/",
+    "start:secure": "webpack-dev-server --https --inline --hot --content-base dist/ --cert ./LocalhostCertificates/localhost.pem --key ./LocalhostCertificates/localhost.key",
+    "start:secure:no-certs": "webpack-dev-server --https --inline --hot --content-base dist/",
```

Comment:
Out of curiosity - did you have to use the version with certs? I usually use a command similar to the no-certs variant.

Reply:
No, I didn't use the version with certs, but added that option in case it was useful in the future.